Core/Src/main.c

```diff
@@ -100,16 +106,18 @@ static void onJoined(EMBENET_PANID panId, const EMBENET_NODE_QuickJoinCredential
 static void onLeft(void) {
     printf("Node has left the network\n");
     // Stop ENMS service
-    EnmsNodeResult enmsStopStatus = ENMS_NODE_Stop(&enmsNode);
-    if (ENMS_NODE_RESULT_OK == enmsStopStatus) {
+    EnmsResult enmsStopStatus = ENMS_NODE_Stop(&enmsNode);
```

Comment:
maybe ignore return value explicitly, since it is inconsistent with other services stopping API. And even if we fail to stop ENMS, the logic stays the same (when we join again, we will try to start it even if it wasn't stopped)